### DISCUSSION TITLE
Can anyone help me deploy this in AWS Cloud.

### DISCUSSION THREAD

**Discussion by aakiliqbal** [charlesbel/Microsoft-Rewards-Farmer#406] [General]
I am new to AWS, I have basic knowledge of the services that are used in AWS. Can anyone help me to deploy in AWS?
Thank You.

**Comment by aliasfoxkde** [comment]
I don't personally think it would be worth hosting on AWS (cost and effort), because you would need a barebone VPS to install Linux on so you can get the repo going. If you have full access to a webhost that has Python access, you could just clone the repo and run it from there, but generally I think that is against most TOC's. They will likely see high CPU usage and flag your account. Not only that, but your IP might be more likely to get flagged as a lot of traffic will be coming from it. Might be better to just get an RPi or something (some have done phones, etc.) and run locally as a bot server. Then just schedule the bots to run daily.

**Comment by aakiliqbal** [comment]
Yeah, I agree with you.
